### PR TITLE
BreadcrumbContent: link and button roles are added

### PR DESCRIPTION
**Pull request by SergeyRoyt**
https://monday.monday.com/boards/2861766393/pulses/3118779705

### CODE REVIEW

File: src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbContent/BreadcrumbContent.jsx [hunk 1]
```diff
@@ -22,7 +22,13 @@ export const BreadcrumbContent = forwardRef(
     if (isClickable && (link || onClick)) {
       if (link) {
         return (
-          <a className={className} href={link} onKeyDown={onKeyDown} aria-current={isCurrent ? "page" : undefined}>
+          <a
+            className={className}
+            href={link}
+            onKeyDown={onKeyDown}
+            aria-current={isCurrent ? "page" : undefined}
+            role="link"
```
Comment (hadasfa) [inline src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbContent/BreadcrumbContent.jsx:30]: when element is a native link (a) there is no need to add also role link

Comment (hadasfa) [reply src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbContent/BreadcrumbContent.jsx:30]: so let's keep the button role and remove this one

Comment (SergeyRoyt) [reply src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbContent/BreadcrumbContent.jsx]: Done

Comment (SergeyRoyt) [reply src/components/BreadcrumbsBar/BreadcrumbItem/BreadcrumbContent/BreadcrumbContent.jsx]: Done

**Pull request merged**